app/renderer/js/main.js

```diff
@@ -142,7 +144,11 @@ class ServerManagerView {
 
 		for (const i in settingOptions) {
 			if (ConfigUtil.getConfigItem(i) === null) {
-				ConfigUtil.setConfigItem(i, settingOptions[i]);
+				if (EnterpriseUtil.isConfigItemExists(i)) {
```

Comment:
Add a comment here stating what this does. I think this initializes default based on enterprise config. If so mention that in a comment.